portal-user-service/src/main/java/com/telecominfraproject/wlan/portaluser/controller/PortalUserController.java

```diff
@@ -148,6 +148,20 @@ public ListOfPortalUsers getAllInSet(@RequestParam Set<Long> portalUserIdSet) {
              throw exp;
         }
     }
+    
+    @RequestMapping(value = "/idsFromUsername", method = RequestMethod.GET)
+    public List<Integer> getIdsfromUsername(@RequestParam String username) {
```

Comment:
typo with lowercase 'f' in `getIdsfromUsername`

Reply:
Fixed in latest